pages/activities/ActivitiesPage.jsx

```diff
@@ -140,14 +139,6 @@ var ActivitiesPage = React.createClass({
               developedBy={<div><a href="http://educatorinnovator.org/">Educator Inovator</a> - <a href="https://twitter.com/jennifer_dick">Jen Dick</a>, <a href="http://technolojie.com/sample-page/">Jie Qi</a>, <a href="https://www.linkedin.com/in/dcole1">David Cole</a>, and <a href="http://twitter.com/chadsansing">Chad Sansing</a></div>}
               description="In this series of activities, learners will explore the connections between art, circuitry and systems thinking by hacking their notbooks/journals with power and LEDs."/>
             <ActivityKit
-              src1x="/img/pages/activities/img-activity-08.jpg"
-              src2x="/img/pages/activities/img-activity-08@2x.jpg"
```

Comment:
These files should be removed too. See https://github.com/mozilla/teach.mozilla.org/pull/1692/files#r55615514